cmd/serverless/main.go

```diff
@@ -116,6 +116,7 @@ func runAgent(stopCh chan struct{}) (serverlessDaemon *daemon.Daemon, err error)
 		log.Debug("Unable to restore the state from file")
 	} else {
 		serverlessDaemon.ComputeGlobalTags(config.GetConfiguredTags(true))
+		serverlessDaemon.StartLogCollection()
```

Comment:
Since `StartLogCollection` is also started after ComputeGlobalTags in `pkg/serverless/serverless.go`, should we move this inside ComputeGlobalTags or inside a method containing both?
Also could we make sure we call this only at one place to avoid the `sync.once` ?

Reply:
I originally had it inside ComputeGlobalTags, but decided to move it out for clarity. The sync.Once would still be necessary because ComputeGlobalTags is called at the start of every invocation.